CMake/FindLua.cmake

```diff
@@ -57,30 +56,30 @@ FIND_LIBRARY (LUA_LIBRARY NAMES ${LUA_LIBRARY_NAME}
   /Library/Frameworks
 )
 
-IF (NOT LUA_INCLUDE_DIR)
-    MESSAGE(FATAL_ERROR "Could Not Find Lua Include Dir")
-ENDIF (NOT LUA_INCLUDE_DIR)
+if(NOT LUA_INCLUDE_DIR)
+  message(FATAL_ERROR "Could Not Find Lua Include Dir")
+endif()
 
-IF (NOT LUA_LIBRARY)
-    MESSAGE(FATAL_ERROR "Could Not Find Lua Library")
-ENDIF (NOT LUA_LIBRARY)
+if(NOT LUA_LIBRARY)
+  message(FATAL_ERROR "Could Not Find Lua Library")
+endif()
 
 
-IF (LUA_INCLUDE_DIR AND LUA_LIBRARY)
-    SET (Lua_FOUND TRUE)
-ENDIF (LUA_INCLUDE_DIR AND LUA_LIBRARY)
+if(LUA_INCLUDE_DIR AND LUA_LIBRARY)
+  set(Lua_FOUND TRUE)
```

Comment:
If `LUA_LIBRARY` or `LUA_INCLUDE_DIR` are not found `FATAL_ERROR` means we bail out. So this check can be removed entirely